lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,215 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
```

Comment:
@mbifulco 

This shouldn't be anonymous, it's causing your Require.js error.  Change this to something like 
`define("gymnasium/js/student_account/views/account_settings_factory", [`

I've only done this using comprehensive theming, and you are using Stanford style but I believe this will work.  It looks like you've already updated the module path in `account_settings.html +1!

cc @tkeemon